modular_splurt/code/game/objects/items/lewd_items/genital_equipment/chastity/chastity_belt.dm

```diff
@@ -0,0 +1,119 @@
+/obj/item/genital_equipment/chastity_cage/belt // I'm making it a genital_equipment, sorry
+	name = "chastity belt"
+	icon = 'modular_splurt/icons/obj/lewd_items/chastity.dmi'
+	icon_state = "belt"
+	slot_flags = ITEM_SLOT_UNDERWEAR
```

Comment:
even if you have the chastity toggles off you can still equip this item in your underwear or others' underwear, it'd be prefrered if you weren't able to equip it at all if the toggles are off

also, for some reason these items can't be deleted with vv